test/fuzz/transformation_record_synonymous_constants_test.cpp

```diff
@@ -90,13 +90,19 @@ TEST(TransformationRecordSynonymousConstantsTest, IntConstants) {
                                                validator_options);
   ASSERT_TRUE(IsValid(env, context.get()));
 
+#ifndef NDEBUG
   // %3 is not a constant declaration
-  ASSERT_FALSE(TransformationRecordSynonymousConstants(3, 9).IsApplicable(
-      context.get(), transformation_context));
+  ASSERT_DEATH(TransformationRecordSynonymousConstants(3, 9).IsApplicable(
+                   context.get(), transformation_context),
+               "The ids must refer to constants.");
+#endif
 
+#ifndef NDEBUG
   // Swapping the ids gives the same result
```

Comment:
It is like that because it checks that giving an id that is not a constant causes an assertion to fail. It is indeed not very clear, having the comment "%3 is not a constant declaration" would probably be better (?).

Reply:
I changed it to "%3 is not a constant declaration"